### PR TITLE
Acceptance testing

### PR DESCRIPTION
**Pull request by earnubs**
add a "happy path" acceptance test for staging that will test:

* login
* stripe.js
* ubuntu payments
* success message in ui

Once this has landed I'll integrate with browserstack, for now you'll need a running mu instance at:
http://localhost:3000

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,6 +81,7 @@
     "babel-preset-stage-0": "^6.16.0",
     "babel-register": "^6.16.3",
     "chokidar": "^1.6.1",
+    "co-mocha": "^1.1.3",
```
Comment (earnubs) [inline package.json]: oops, this wasn't used, i discovered that webdriver has it's own wrapper for mocha that uses generators

File: package.json [hunk 2]
```diff
@@ -93,12 +95,14 @@
     "mocha": "^3.1.2",
     "nock": "^9.0.2",
     "nyc": "^9.0.1",
+    "phantomjs-prebuilt": "^2.1.13",
```
Comment (earnubs) [inline package.json]: this _also_ should go into dependencies-devel

File: src/common/components/customer-success/index.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import tick from '../../style/tick.css';
 export default function CustomerSuccess() {
   return (
     <div className={ styles.customerSuccess }>
-      <h3 className={ tick.green }>Payment details successfully saved, thank you!</h3>
+      <h3 data-qa="customer-success:thanks" className={ tick.green }>Payment details successfully saved, thank you!</h3>
```
Comment (earnubs) [inline src/common/components/customer-success/index.js]: use @data to reduce fragility in tests

File: README.md [hunk 4]
```diff
@@ -43,6 +43,8 @@ Have you created your own .env file that is useful for a common development situ
 See [Environment Variables](docs/environment-variables.md).
 
 
-# Why javan-rhino?
+## Acceptance testing
 
-Win a prize if you figure it out!
+```
```
Comment (bartaz) [inline README.md]: Please add a note about having local server running in background when testing locally with phantom.

**Comment by earnubs** [comment]
[E2E Tests](https://trello.com/c/a9Fo4goC/79-e2e-tests)

**Pull request merged**